hhs_oauth_server/settings/aws-impl.py

```diff
@@ -40,9 +40,22 @@
 IMPL_SPECIFIC_APPS = [
     # Installation/Site Specific apps based on  -----------------
     'storages',
+    # A test client - moved to aws-test / dev /impl settings
+    'apps.testclient',
```

Comment:
Impl will soon be our sandbox environment, where developers can play with synthetic data. Let's try to keep these types of internal testing tools out of it? Quick summary:

* Dev isn't internal right now, but will be after we update our URLs
* Test is internal
* Impl is external (although maybe not right now)
* Prod is external